src/main/java/org/folio/innreach/batch/contribution/service/ContributionJobRunner.java

```diff
@@ -472,20 +493,22 @@ private void runOngoing(UUID centralServerId, BiConsumer<ContributionJobContext,
       endContributionJobContext();
     }
     catch (ServiceSuspendedException | FeignException | InnReachConnectionException | SocketTimeOutExceptionWrapper e) {
-      log.info("exception thrown from runOngoing");
+      log.info("exception thrown from runOngoing :", e);
       throw e;
     }
     catch (Exception e) {
-      log.info("contributeInstance exception block :{}",e.getMessage());
+      log.info("contributeInstance exception block :",e);
       throw e;
     }
   }
 
   private boolean isUnknownEvent(InstanceIterationEvent event, UUID iterationJobId) {
+    log.info("isUnknownEvent:: parameters event: {}, iterationJobId: {}", event, iterationJobId);
```

Comment:
This will be very misleading in the logs. It's not that useful to know if a method was called since it already is guranteed to be called. Instead we only want to know if an unknown event occurs. So you should only log it if what is returned is true, and that can be done where `isUnknown` event is called above.

The reason it is misleading is because if you saw this in the logs wouldn't you think that it is an unknown event that was encountered. I would until I looked at the source...